### PR TITLE
fixed C++11

### PR DESCRIPTION
**Pull request by cielavenir**

### CODE REVIEW

File: src/binpickingtask.cpp [hunk 1]
```diff
@@ -296,7 +296,7 @@ std::string utils::GetJsonString(const BinPickingTaskResource::DetectedObject& o
 std::string utils::GetJsonString(const BinPickingTaskResource::PointCloudObstacle& obj)
 {
     std::stringstream ss;
-    ss << std::setprecision(std::numeric_limits<decltype(obj.points)::value_type>::digits10+1);
+    ss << std::setprecision(std::numeric_limits<float>::digits10+1);
```
Comment (woodychow) [inline src/binpickingtask.cpp]: The whole point of this is whenever the type of BinPickingTaskResource::PointCloudObstacle changes, this code won't cause any problem. If you really have to change it....I would say use ifdefs.

Comment (akovski) [reply src/binpickingtask.cpp]: you can use rapidjson to avoid these stringstream

Comment (cielavenir) [reply src/binpickingtask.cpp]: @akovski good idea. Could we handle NaN with rapidjson?

Comment (akovski) [reply src/binpickingtask.cpp]: no...

Comment (cielavenir) [reply src/binpickingtask.cpp]: I'm not sure about the intention of commit https://github.com/mujin/controllerclientcpp/commit/29123635d9a3e39741b70b8cc0a327fa1ddaf8cf , but perhaps C++98 people tend to typedef pointcloud type. `typedef float pointfloat_t;`

Comment (woodychow) [reply src/binpickingtask.cpp]: Regarding that commit, dReal is double and the points from the sensors are in float. We were wasting bandwidth / time to convert between double/float.

**Comment by rdiankov** [comment]
fixed in master

**Pull request closed without merging**